fisheye:
   imgWidth: 1920
   imgHeight: 1080
   fov_camera: 156
   fov_center_roi: 156
   cam_file: "/home/mchris/buffer/pmtd_stitching_data_20230515_1830/intrinsic_cam02.yaml"
   undistort_scale: 4

test:
   image_src_path: "/home/mchris/catkin_workspace/mei_camera_tool/src/image_undistort_tool/data/undistort_image/left0002.jpg"
   image_dst_path: "/home/mchris/catkin_workspace/mei_camera_tool/src/image_undistort_tool/data/undistort_image/left_test_"
